Tests/ComposedUITests/ChangesReducerTests.swift

```diff
@@ -0,0 +1,19 @@
+import XCTest
+import Composed
+@testable import ComposedUI
+
+final class ChangesReducerTests: XCTestCase {
+    func testGroupInsertsUpdates() {
+        var changesReducer = ChangesReducer()
+
+        changesReducer.insertGroups(IndexSet([0, 1, 2]))
+
+        XCTAssertEqual(changesReducer.changeset.groupsInserted, [0, 1, 2])
+        XCTAssertTrue(changesReducer.changeset.groupsRemoved.isEmpty)
+        XCTAssertTrue(changesReducer.changeset.groupsUpdated.isEmpty)
+        XCTAssertTrue(changesReducer.changeset.elementsRemoved.isEmpty)
+        XCTAssertTrue(changesReducer.changeset.elementsInserted.isEmpty)
+        XCTAssertTrue(changesReducer.changeset.elementsUpdated.isEmpty)
+        XCTAssertTrue(changesReducer.changeset.elementsMoved.isEmpty)
+    }
+}
```

Comment:
I assume you plan on adding more test cases later on?

Reply:
This is only the start -- the `CollectionCoordinator` doesn't use it yet.

More tests will be added to this file before merging, but I think it's going to lean a little on the new `CollectionCoordinator` tests passing to validate correctness according to `UICollectionView`.